test/utils/initMatchers.ts

```diff
@@ -271,6 +291,34 @@ chai.use((chaiAPI, utils) => {
     // eslint-disable-next-line no-underscore-dangle, @typescript-eslint/no-unused-expressions
     new chai.Assertion(this._obj).to.be.visible;
   });
+
+  chai.Assertion.addMethod('toIncludeStyle', function toHaveComputedStyle(
+    expectedStyleUnnormalized: Record<string, string>,
+  ) {
+    // Compare objects using hyphen case.
+    // This is closer to actual CSS and required for getPropertyValue anyway.
+    const expectedStyle: Record<string, string> = {};
+    Object.keys(expectedStyleUnnormalized).forEach((cssProperty) => {
+      expectedStyle[_.kebabCase(cssProperty)] = expectedStyleUnnormalized[cssProperty];
+    });
+
+    // eslint-disable-next-line no-underscore-dangle
+    const style = this._obj as CSSStyleDeclaration;
+    const actualStyle: Record<string, string> = {};
+    Object.keys(expectedStyle).forEach((cssProperty) => {
+      actualStyle[cssProperty] = style.getPropertyValue(cssProperty);
+    });
+
+    const assertion = new chai.Assertion(
+      actualStyle,
+      'Styles did not match. ' +
+        'Test results in JSDOM e.g. from `test:unit` are often misleading since JSDOM does not implement the Cascade nor actual CSS property value computation. ' +
+        'If results differ between real browsers and JSDOM, skip the test in JSDOM e.g. `if (/jsdom/.test(window.navigator.userAgent)) this.skip();`',
```

Comment:
Just wondering, should we always skip this kinds of tests in JSDOM, or should we maybe mention here in the message which css properties are good for testing in JSDOM (like avoid colors, use margin, padding instead etc..)

Reply:
Alright, makes sense to me. Thanks for the clarification (I've already spent an hour so far debugging this, so seeing an error like this would definitely help me at least :))